qa/rpc-tests/bip135-grace.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python3
```

Comment:
do not use "-" in python file names.  This means they can't be used interactively:
$ python
>>> import bip135-grace
  File "<stdin>", line 1
    import bip135-grace
                          ^
SyntaxError: invalid syntax

interactive use is nice for development and debugging.

Reply:
ok, files will be renamed